dev/scripts/src/alluxio.org/build-distribution/cmd/flags.go

```diff
@@ -78,24 +70,27 @@ func handleUfsModulesAndLibJars() error {
 			}
 		}
 	}
-	if strings.ToLower(includedLibJarsFlag) == "all" {
+	switch strings.ToLower(includedLibJarsFlag) {
+	case libJarsAll:
 		var allLibJars []string
 		for jar := range coreLibJars {
 			allLibJars = append(allLibJars, jar)
 		}
-		for jar := range libJars {
+		for jar := range additionalLibJars {
 			allLibJars = append(allLibJars, jar)
 		}
 		includedLibJarsFlag = strings.Join(allLibJars, ",")
-	} else if strings.ToLower(includedLibJarsFlag) == "core" {
+	case libJarsCore:
 		var coreJars []string
 		for jar := range coreLibJars {
 			coreJars = append(coreJars, jar)
 		}
 		includedLibJarsFlag = strings.Join(coreJars, ",")
-	} else {
+	default:
 		for _, jar := range strings.Split(includedLibJarsFlag, ",") {
-			if _, ok := libJars[jar]; !ok {
```

Comment:
this was a bug since it was only iterating through the additional jars, missing the core ones

Reply:
Good catch!